packages/browser/core/src/core/auth/StitchAuthListener.ts

```diff
@@ -30,6 +59,16 @@ export default interface StitchAuthListener {
    * * When a user is linked to another identity.
    * * When a listener is registered. This is to handle the case where during registration an event happens that the registerer would otherwise miss out on.
    *
+   * The [[StitchAuth]] instance itself is passed to this callback. This can be used to read the current state of authentication.
+   *
+   * ### Note
+   * Specific event details are deliberately not provided here because the events could be stale by the time they are handled.
+   * 
+   * For example, a user could log in then log out before the first login event is handled.
+   * 
+   * The intention is that you would treat this callback as a trigger to refresh the relevant parts of your app based
+   * on the new, current auth state. In other words, it should not be necessary to deduce exactly what changed in this callback.
```

Comment:
not sure if I agree with the "In other words, it should not be necessary to deduce exactly what changed". That's ultimately up to the app developer. If they want to maintain previous state so that they can compare to find out exactly what happened, that's totally fine. It might even help optimize whatever state they are refreshing.

Reply:
Yeah, I think I went overly opinionated with that line, when what I really wanted to convey is already covered by the first line.